src/mitigation.c

```diff
@@ -54,18 +56,37 @@ void mitigation_add_resource(struct mitigation *m, const char *name, const char
 
 	r->resource = res;
 
-	list_append(&m->resources, &r->list_node);
+	// resources that have the attribute fallback will be added to a seperate list
+	if(fallback)
```

Comment:
I opted for a valueless attribute,where the existence of the attribute is itself sufficient,  instead of a boolean. But sure, will use a boolean logic in a later revision.